### ISSUE TITLE
Full Version Checkpoint Required for Fine-Tuning

### ISSUE THREAD

**Issue by nku-zhichengzhang** [hkchengrex/Tracking-Anything-with-DEVA#103]
Hi,
Thanks for the great work. I am currently working on finetuning the DEVA model for the downstream application and I have encountered an issue with the provided checkpoint. The current checkpoint seems to be missing several components including the iteration, network, optimizer, and scheduler. Could you kindly provide a full version of the checkpoint that includes these components?

![image](https://github.com/hkchengrex/Tracking-Anything-with-DEVA/assets/44353451/a69e0bb3-1241-43eb-9fdd-9273f99139a8)

**Comment by hkchengrex** [comment]
https://drive.google.com/file/d/1TiqUf3gI2gxdyQByFNUZTIk2irw7AG3H/view?usp=sharing

Note that this might not be the exact same training run as the released weights, but it should be good enough for finetuning.
